packages/eslint-config/index.js

```diff
@@ -121,27 +114,19 @@ module.exports = [
     files: ["**/*.ts"],
     ignores: ["**/*.d.ts"],
     plugins: {
-      ts: typescriptPlugin,
+      "@typescript-eslint": typescriptPlugin,
```

Comment:
The new Prettier ESLint config supports the new config file format but expects `@typescript-eslint` not to be aliased as it overrides some of its rules.